docs/additional/interesting_aspects_de.md

```diff
@@ -0,0 +1,248 @@
+
+[strict-mode]: http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+#### Line-Endings
+
+Für alle Skripte sollten jederzeit Unix-basierte Line-Endings benutzt werden.
+
+#### Doguctl verwenden
+
+Doguctl ist ein Kommandozeilentool, um die Konfiguration eines Dogus zu vereinfachen.  
+Informationen über die Benutzung finden sich in ["Die Nutzung von doguctl"][doguctl-usage].
+
+Wo anwendbar, soll immer Doguctl in Bash-Skripten wie der `startup.sh` verwendet werden.  
+Für die sinnvolle Benutzung kann das [Startup-Skript vom Nexus][doguctl-example] als Beispiel genommen werden.
+
+[doguctl-usage]: ../important/relevant_functionalities_de.md#die-nutzung-von-doguctl
+[doguctl-example]: https://github.com/cloudogu/nexus/blob/4d2de3733eca684df1363179c527363a4d31526e/resources/startup.sh
+
+### Dogu-Startskript: startup.sh
+
+Das [Dogu-Start-Skript][startup-sh] ist der Einstiegspunkt eines Dogu-Containers und verwaltet alle Schritte, die für einen reibungslosen Start des Dogus notwendig sind.
+Konventionell wird das Startup-Skript mit dem Namen `startup.sh` in dem `resources`-Ordner des Dogus angelegt.
+
+[startup-sh]: ../important/relevant_functionalities_de.md#aufbau-und-best-practices-von-startupsh
+
+## Schnelle Feedback-Zyklen durch Tool-Entwicklung ohne CES
+
+In der Entwicklung ist eine rasche Erkenntnis wichtig, ob der richtige Entwicklungspfad beschritten wurde.  
+Ein schnelles Feedback ist umso wichtiger, je komplexer die Umgebung wird.
+
+In der Dogu-Entwicklung lässt sich schnelles Feedback auf unterschiedliche Weise erreichen:
+
+### Multi-Stage-Build
+
+Für eine effektive Entwicklung wird der Einsatz eins Multi-Stage-Builds für das Dogu-Image empfohlen.
+Diese bieten Vorteile wie ein effektiveres Caching von Downloads, schnellere Bauzeiten, effektives Layering und kleine Image-Größen.
+Mehr Information zu Multi-Stage-Build gibt es auf der [offiziellen Website von Docker][docker-multi-stage].
+
+[docker-multi-stage]: https://docs.docker.com/develop/develop-images/multistage-build/
+
+### CES-freie Entwicklungsumgebung
+
+Das Bauen des Dogus und das Ausliefern ins lokale EcoSystem fordert nicht nur viel Zeit, sondern nimmt auch die Möglichkeit die Software effektiv zu debuggen.  
+Daher ist es sinnvoll eine lokale Entwicklungsumgebung unabhängig vom CES aufzubauen, um die Dogu-Software effektiv auszuführen, zu testen und zu debuggen.
+
+Die lokale Entwicklungsumgebung muss alle abhängigen CES-Dienste durch lokal gestartete Alternativen zu Verfügung stellen.
+Wir empfehlen, dass eine lokale Entwicklungsumgebung in Form einer `docker-compose.yml`-Datei definiert ist und jederzeit einfach mit `docker-compose up` gestartet und mit `docker-compose down` heruntergefahren werden kann.  
+Für ein Beispiel [siehe CAS][local-cas-example].
+
+[local-cas-example]: https://github.com/cloudogu/cas/blob/develop/app/docker-compose.yml
+
+## Qualitätssicherung für Dogus
+
+### Container Validation (GOSS)
```

Comment:
Throwing `(GOSS)` in the title does not really fit into a readible narative. Also, Goss names itself with an uppercase G letter. I suggest to either remove the application `Container Validation` or rephrase it `Container Validation with Goss`